app/frontend/src/components/loader/loader.scss

```diff
@@ -1,10 +1,9 @@
-.js-location-finder {
-
+.location-finder {
```

Comment:
wasn't it a good practice to prefix with js classes that were affected by javascript changes?

Reply:
ok funny i was just talking to connor about this

Reply:
is this good practice in the alphagov guidelines? i happy either way as long as consistent

Reply:
actually i will revert this to keep the js- and keep it out of this task because there is js- in many other places